F 1.22 updates

kylebmeredith opened the pull request:
## Story/Task
Update the docs for 1.22

## Value
## To-do
## Notes

kylebmeredith committed:
Add audio data textures

kylebmeredith committed:
Add external libraries page

kylebmeredith committed:
Update quickstart guide

kylebmeredith committed:
Update media uniforms

kylebmeredith committed:
Add wrap and filter options to JSON config

kylebmeredith committed:
Update functions

kylegrover commented on docs/ssf/audio_uniforms.md:
```diff
@@ -2,7 +2,24 @@
+
+There are two textures which store different audio data in each channel. Here is a visualization of these 2DSamplers from the built-in Audio Indicators scene:
+
+[![Audio Samplers](../src/images/audio_samplers.gif)](../src/images/audio_samplers.gif)
```
I like the idea of adding more video clips, might be better off mp4s though

kylegrover commented on docs/ssf/libraries.md:
```diff
@@ -1,15 +1,76 @@
+To learn more about the library and all its functions/files, check out the [github](https://github.com/patriciogonzalezvivo/lygia) or [website](https://lygia.xyz/). Here are links to documentation for each section of the library (copied from the website):
+
+<ul>
+	<li><a href="https://lygia.xyz/math"><code>math/</code></a>: general math functions and constants: <code>PI</code>, <code>SqrtLength()</code>, etc. </li>
```
we should lock in what version of Lygia we're using and make a note of it here. we can also link to that version of their docs ie `https://lygia.xyz/v1.1.4/math`

kylegrover commented on docs/ssf/libraries.md:
```diff
@@ -1,15 +1,76 @@
+
+## hg_sdf
+
+[hg_sdf](https://mercury.sexy/hg_sdf/) is an open-source library that provides a powerful toolset for creating raymarched scenes with signed distance functions (sdf). The following documentation is provided by the Mercury team. For more information, check out [their website](https://mercury.sexy/hg_sdf/).
```
this site still gets blocked for me, figured out that its my centurylink router's 'cybersecurity' feature

kylegrover commented on docs/ssf/standard_uniforms.md:
```diff
@@ -38,5 +38,5 @@ Uniform | Description
-`sampler2D syn_UserImage` | The texture of the current media selected in the media section of the control panel. **NOTE:** to load this texture into your scene, you should use the [`loadUserImage()` function](functions.md#_loaduserimage) or [`textureUserImage()` function](functions.md#_textureuserimage)
-`float syn_MediaType` | Indicates the type of the current media: `None = 0`, `Image = 1`, `Video = 2`, `Webcam = 3`. You can check if media is loaded with `syn_MediaType > 0.5`.
\ No newline at end of file
+`sampler2D syn_Media` | The texture of the current media selected in the media section of the control panel. **NOTE:** to load this texture into your scene, you should generally use the [`loadMedia()` function](functions.md#_loadMedia) or [`textureMedia()` function](functions.md#_texturemedia)
```
should we include a note on syn_UserImage?

kylegrover reviewed (approved):
looks awesome, I would've been so juiced to have all of this info laid out for me when I started

kylegrover commented on docs/ssf/audio_uniforms.md:
```diff
@@ -2,7 +2,24 @@
+
+### Data Textures
+
+There are two textures which store different audio data in each channel. Here is a visualization of these 2DSamplers from the built-in Audio Indicators scene:
```
1DSamplers?

kylebmeredith committed:
Minor updates

- add new installer links
- add version to lygia
- remove outdated links
- 2D -> 1D
